dotnet/src/SemanticKernel/CoreSkills/HttpSkill.cs

```diff
@@ -62,9 +66,10 @@ public HttpSkill() : this(new HttpClient(s_httpClientHandler, disposeHandler: fa
     /// <param name="context">Contains the body of the request</param>
     /// <returns>The response body as a string.</returns>
     [SKFunction("Makes a POST request to a uri")]
+    [SKFunctionContextParameter(Name = "serializedHeaders", Description = "The HTTP request headers dictionary serialized using System.Text.JSON")]
```

Comment:
Hey @sus19agg, thank you for your contribution to extending and improving SK. I wanted to bring your attention to the [updated version of the HttpSkill](https://github.com/microsoft/semantic-kernel/blob/4b0f04fe9cc090eb90c081f40871ab5dd222b84d/dotnet/src/SemanticKernel/CoreSkills/HttpSkill.cs) in [this PR](https://github.com/microsoft/semantic-kernel/pull/1195). It no longer relies on SKContext for argument resolution and instead accepts all arguments through method parameters. It would be really helpful if you could wait until that PR is merged and align your solution with the one proposed there.
CC: @stephentoub

Reply:
Hi @SergeyMenshykh, Thank you for bringing this to my attention. As suggested, I will wait for the mentioned PR to merge and align my solution accordingly.